### DISCUSSION TITLE
Background image opacity is not working

### DISCUSSION THREAD

**Discussion by PriyanshuYadavksolves** [tailwindlabs/tailwindcss#13456] [Help]
i m not able to set opacity of background image, can anyone solve it i dont want to include vanilla css in my file to manupulate opacity

**Comment by alex-krasikau** [comment]
Hey @PriyanshuYadavksolves!

CSS doesn't directly support background image opacity. One way to work around this is by using an absolutely positioned `<img />` element as a background. This method provides greater flexibility than the traditional `background-image` property. Compared to `background-image`, `<img />` offers much better image optimization capabilities.

Here's an example solution:
```html
<div class="relative">
  <img 
    class="absolute inset-0 h-full w-full max-w-none object-cover opacity-30" 
    src="{image url}" 
  />

  <div class="isolate">
     {content}
  </div>
</div>
```

You can check out the complete example here: [Tailwind Labs - Background Opacity](https://play.tailwindcss.com/jHiUyRxJ0f).

Alternatively, you can use the `:before` pseudo-element trick, which functions similarly but is more verbose and limiting.

**Reply by PriyanshuYadavksolves** [reply]
Thanks @alex-krasikau for the help
